decisions/2021/03/18-large-nested-collections.md

```diff
@@ -0,0 +1,40 @@
+JIRA:
+
+-   DHIS2-10699
+
+Use cases:
+
+1.  Browsing and editing in bulk
+2.  Finding and changing a single option
```

Comment:
This wasn't directly mentioned in the meeting, but this second use-case does not really fit with the "atomic updates" approach. Since that approach has some significant advantages (creating/updating in a single request), I think we should probably omit this second use-case. If we do need functionality for that, a create/edit form is probably not exactly the right place. We're probably best of keeping the second use-case out-of-scope for now.

Reply:
I think we covered this as far as options as collection in another object goes. We did mention it as an example but then talked more general.

Reply:
Not sure what to change here, so will leave it as is. Even if we omit it for now, it is a use case that can be solved by involving UX and a different view. The use case doesn't state that it must be in the create/edit form.